configuration/amd64/mysql/add_v2xhub_user.sh

```diff
@@ -2,8 +2,8 @@
 # Fail when any command fails
 #set -e
 # Ensure mysql-client is installed
-sudo apt-get update
-sudo apt-get install mysql-client
+#sudo apt-get update
```

Comment:
remove comment out code

Reply:
Didn't mean to make this change. Will undo